### ISSUE TITLE
Building OpenCPU - dpkg-buildpackage throwns an error about GPG

### ISSUE THREAD

**Issue by alliquot** [jeroenooms/opencpu#172]
Today I tried to install OpenCPU on a new machine with the lastest stable Debian (64 bit).

I downloaded the tarball, unpacked it and run dpkg-buildpackage. At the end of the process it thrown an error: 

    wget https://github.com/jeroenooms/opencpu-server/archive/v1.5.tar.gz
    tar xzf v1.5.tar.gz
    cd opencpu-server-1.5
    dpkg-buildpackage

    dpkg-buildpackage: full upload; Debian-native package (full source is included)
     signfile opencpu_1.5.2-trusty8.dsc
    gpg: skipped "Jeroen Ooms <jeroen@opencpu.org>": No secret key
    gpg: dpkg-sign.5RBlcV8R/opencpu_1.5.2-trusty8.dsc: clearsign failed: No secret key

    dpkg-buildpackage: error: failed to sign .dsc and .changes file

How can I solve it?

EDIT:
    apt-get install software-properties-common
    sudo add-apt-repository ppa:opencpu/opencpu-1.5 -y
    sudo apt-get update

didn't work:
    W: Failed to fetch http://ppa.launchpad.net/opencpu/opencpu-1.5/ubuntu/dists/jessie/main/binary-amd64/Packages  404  Not Found

EDIT:
I've found the following location on the server:
    http://ppa.launchpad.net/opencpu/opencpu-1.5/ubuntu/dists/trusty/main/debian-installer/binary-amd64/

So I went to the sources.list.d, opened opencpu....jessie.list and replaced _jessie_ with _trusty_. Then _apt-get install opencpu_ worked.

I don't know which repository has been used: Ubuntu/trusty or debian?

Will the installation prepared for Ubuntu work well for Debian? I would really like build it from sources, but couldn't get it done.

I saw that it failed at "apparmor" starting, but OpenCPU seems to be working now. I can run examples in my webbrowser. This is very important installation for me so I have to be sure it can be installed this way (apt-get from Ubuntu repo) on Debian.

**Comment by jeroenooms** [comment]
You probably should not install packages that were built for another distribution. You might run into version conflicts.

Can you try using `dpkg-buildpackage -us -uc` to create build the package without signing it? Check `dpkg-buildpackage --help` for other options.

**Comment by alliquot** [comment]
Yes, it works perfectly.

I was able to create all *.deb files and install them successfully. And while I installed them on a working instance of OpenCPU (just stopped both Apache and Ocpu services before), which caused all files to be overwritten, everything works perfectly.

I can hereby confirm that the Ubuntu package works perfectly on Debian Jessie (8).
Thus, It seems it can be installed both from the Ubuntu repository or by building them by dpkg-buildpackage.

Thank you very much Jeroen :)

**Comment by alliquot** [comment]
Yes, it works perfectly.

I was able to create all *.deb files and install them successfully. And while I installed them on a working instance of OpenCPU (just stopped both Apache and Ocpu services before), which caused all files to be overwritten, everything works perfectly.

I can hereby confirm that the Ubuntu package works perfectly on Debian Jessie (8).
Thus, It seems it can be installed both from the Ubuntu repository or by building them by dpkg-buildpackage.

Thank you very much Jeroen :)
